k8s/crd-samples/pool-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorpools.openebs.io
+  namespace: default
```

Comment:
remove the namespace, since this is at Cluster scope

Reply:
ok